jetson/face_detector_threaded.py

```diff
@@ -293,17 +296,18 @@ def encryptFrame(img, boxes):
     g.eval()
     class_names = ['Glasses', 'Goggles', 'Neither']
 
-    cap = VideoCapturer() #Instantiate Video Capturer object
-    detector = FaceDetector(trained_model=args.trained_model, cuda=args.cuda and torch.cuda.is_available(), set_default_dev=True) #Instantiate Face Detector object
-    cl = Classifier(g) #Instantiate Classifier object
+    cap = VideoCapturer()  # Instantiate Video Capturer object
+    detector = FaceDetector(trained_model=args.trained_model, cuda=args.cuda and torch.cuda.is_available(),
+                            set_default_dev=True)  # Instantiate Face Detector object
+    cl = Classifier(g)  # Instantiate Classifier object
```

Comment:
the comment "instantiate <object name> object" is useless to a developer. the code clearly shows that's happening. Comments are meant to tell developers something the code can't like "this line is needed to ensure this runs properly etc ...". In general your comment should: tell me why you need to instantiate this class and what its used for. that is information that line of code doesn't convey. explicitly but will be helpful to future developers

Reply:
Could we just remove the comments? Once the classes are documented it will be obvious what each instantiation does.

Reply:
yes just removing them is fine